install_vendored_static_library_xcframework/after/BananaLib/BananaLib.podspec

```diff
@@ -0,0 +1,31 @@
+
+Pod::Spec.new do |spec|
+  spec.name         = "BananaLib"
+  spec.version      = "0.0.1"
+  spec.summary      = "Amazing bananalib that provides access to bananas"
+
+  spec.description  = <<-DESC
+  Amazing bananalib that provides access to bananas
+                   DESC
+
+  spec.homepage     = "http://github.com/CocoaPods/CocoaPods"
+  spec.license      = "MIT"
+
+  spec.author       = { "CocoaPods" => "someone@cocoapods.org" }
+
+  spec.platform     = :ios, "13.0"
+
+  spec.source       = { :git => "https://github.com/CocoaPods/CocoaPods.git", :tag => "#{spec.version}" }
+  spec.source_files  = "Classes", "Classes/**/*.{h,m}"
+  spec.exclude_files = "Classes/Exclude"
+
+  spec.vendored_frameworks = 'CoconutLib.xcframework'
+
+  spec.pod_target_xcconfig = {
+    'HEADER_SEARCH_PATHS' => '$(inherited) "${PODS_XCFRAMEWORKS_BUILD_DIR}/CoconutLib/Headers"',
```

Comment:
this shouldn't be necessary as it is supposed to be handled here: https://github.com/CocoaPods/CocoaPods/blob/44243c7d96009157b590962324a1e3d405178ff4/lib/cocoapods/target/build_settings.rb#L1175-L1180

Reply:
Oh interesting... Ahh, that's only for the aggregate target, there's nothing similar happening for the target of the vending pod. That explains some things.

e.g.: 
- BananaLib pod includes CoconutLib as a vendored xcframework.
- BananaLib contains code that imports and calls CoconutLib.
- but BananaLib can't actually see CoconutLib unless HEADER_SEARCH_PATHS is modified as above, because `pod install` isn't automatically adding that.

I assume now that this is a bug. I'll update my existing PR with a fix for this as well, and then... well, this test case will also cover it once the above is removed from the podspec.

Reply:
Updated to remove HEADER_SEARCH_PATHS from the podspec (assuming the corresponding fix I added in https://github.com/CocoaPods/CocoaPods/pull/10072 is acceptable).

Reply:
is this resolved/handled now?

Reply:
Yes